src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movies } from '../../react-app-env';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface Props {
+  movies: Movies;
+}
+
+export const MovieCard: React.FC<Props> = ({ movies }) => (
```

Comment:
use destructuring here, it's more comfortable

Reply:
agree, will be better to use destructuring for getting main fields from movie